qbft/instance.go

```diff
@@ -57,50 +57,50 @@ func (i *Instance) Start(value []byte, height Height) {
 			if err != nil {
 				// TODO log
 			}
-			if err := i.Broadcast(proposal); err != nil {
+
+			proposalEncoded, err := proposal.Encode()
+			if err != nil {
+				return
+			}
+
+			msgID := types.PopulateMsgType(i.State.ID, types.ConsensusProposeMsgType)
+
+			broadcastMsg := &types.Message{
+				ID:   msgID,
+				Data: proposalEncoded,
+			}
+
+			if err := i.Broadcast(broadcastMsg); err != nil {
 				// TODO - log
 			}
 		}
 	})
 }
 
-func (i *Instance) Broadcast(msg *SignedMessage) error {
```

Comment:
removing this func means we duplicate the broadcast func everywhere.. any reason we removed it?